hedera-node/hedera-file-service/src/main/java/module-info.java

```diff
@@ -3,6 +3,7 @@
 
     uses com.hedera.node.app.service.file.FileService;
 
+    requires com.swirlds.platform.core;
```

Comment:
Inversion of control typically would dictate that this API module should only have a dependency on another API module. Is that not possible here (or in other module-info files from this PR)?

Reply:
Fair enough. I moved the classes to `swirlds-state-api` module instead.